src/Clay/Common.hs

```diff
@@ -15,17 +15,21 @@ import Data.String (IsString)
 
 -------------------------------------------------------------------------------
 
-class All      a where all      :: a
-class Auto     a where auto     :: a
-class Baseline a where baseline :: a
-class Center   a where center   :: a
-class Inherit  a where inherit  :: a
-class None     a where none     :: a
-class Normal   a where normal   :: a
-class Visible  a where visible  :: a
-class Hidden   a where hidden   :: a
-class Initial  a where initial  :: a
-class Unset    a where unset    :: a
+class All        a where all        :: a
+class Auto       a where auto       :: a
+class Baseline   a where baseline   :: a
+class Center     a where center     :: a
+class Inherit    a where inherit    :: a
+class None       a where none       :: a
+class Normal     a where normal     :: a
+class Visible    a where visible    :: a
+class Hidden     a where hidden     :: a
+class Initial    a where initial    :: a
+class Unset      a where unset      :: a
+class MinContent a where minContent :: a
```

Comment:
Is there any chance that some other types than `Value` will also be instances of your new classes?